packages/nextjs/components/ProductSort.tsx

```diff
@@ -2,8 +2,21 @@ import * as React from "react";
 import { PAGE_QUERY_PARAM, SORT_OPTIONS, SORT_OPTIONS_ARRAY, SORT_QUERY_PARAM, SortType } from "utils/sorting";
 import { useRouterQueryParams } from "utils/useRouterQueryParams";
 import { Pill } from "./Pill";
+import { Select } from "./Select";
 
-export const ProductSort: React.FC = () => {
+type ProductSortProps = {
+  as?: "select" | "pills";
+  showTitle?: boolean;
+  title?: string;
+  selectClassName?: string;
+};
+
+export const ProductSort: React.FC<ProductSortProps> = ({
+  as = "pills",
+  showTitle = true,
```

Comment:
WDYT about making this `showTitle` prop `false` by default, it makes it a little clearer when seeing this component being used elsewhere when we have to explicitly add `showTitle` to the props

Reply:
Yeah, sounds good, actually I was thinking to get rid of `showTitle` and just to validate `title.length`

Reply:
Done